src/amuse/support/argparse.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+
+
+def new_argument_parser(
+    **kwargs
+):
```

Comment:
If there were a docstring here that could get parsed by the documentation generator, then it would be easier for people to understand how to call this. `**kwargs` isn't the clearest parameter declaration :smile:.

Reply:
Absolutely. 
Just felt I had to split these two functions in order to make the arguments available without the need for a new parser. 
But that could probably be solved in a more practical way.